types/ali-oss/index.d.ts

```diff
@@ -990,6 +990,11 @@ declare class OSS {
      */
     list(query: OSS.ListObjectsQuery | null, options: OSS.RequestOptions): Promise<OSS.ListObjectResult>;
 
+    /**
+     * List Objects in the bucket.(V2)
+     */
+    listV2(query: OSS.ListV2ObjectsQuery | null, options: OSS.RequestOptions): Promise<OSS.ListObjectResult>;
```

Comment:
Should `options` be optional? There's a default value in the implementation: https://github.com/ali-sdk/ali-oss/blob/a759699be8c7e4acbdc72a5f8d121f2224a7f846/lib/object.js#L243, and the readme says it's optional.

Reply:
yes, you are right, thanks.